### PR TITLE
feat: Make qualtrics configurable for the user (GH-4424)

### PR DESCRIPTION
**Pull request by bgambocjaviniar**
Closes GH-4424

### CODE REVIEW

**Comment by znikola** [comment]
As discussed, please create the PR against an epic branch.
Also, trigger the jenkins build.

File: projects/core/src/qualtrics/services/qualtrics-config.service.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+export class QualtricsConfigService {
+  qualtrics = this.winRef.nativeWindow['QSI'];
+
+  constructor(private winRef: WindowRef, private config: QualtricsConfig) {
+    if (this.qualtrics && !this.config.qualtrics.multi) {
+      this.reloadCreative();
+    }
+  }
+
+  trigger() {
```
Comment (znikola) [inline projects/core/src/qualtrics/services/qualtrics-config.service.ts]: Please specify a return type.

File: projects/core/src/qualtrics/services/qualtrics-config.service.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      if (this.config.qualtrics.multi) {
+        this.qualtrics.API.unload();
+        this.evaluateSurveyOnce();
+      } else {
+        this.evaluateSurveyOnce();
+      }
+    }
+  }
+
+  private reloadCreative() {
```
Comment (znikola) [inline projects/core/src/qualtrics/services/qualtrics-config.service.ts]: Please specify a return type.

File: projects/core/src/qualtrics/services/qualtrics-config.service.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    this.qualtrics.API.unload();
+  }
+
+  private evaluateSurveyOnce() {
```
Comment (znikola) [inline projects/core/src/qualtrics/services/qualtrics-config.service.ts]: Please specify a return type.

File: projects/storefrontapp/src/index.html [hunk 4]
```diff
@@ -81,6 +81,11 @@
       src="webApplicationInjector.js"
       data-smartedit-allow-origin="localhost:9002"
     ></script>
+
+    <!--BEGIN QUALTRICS WEBSITE FEEDBACK SNIPPET-->
```
[the remark this answers is not in the document]
Comment (bgambocjaviniar) [reply projects/storefrontapp/src/index.html]: its copy pasted from the qualtrics web app's deployment code

File: projects/core/src/qualtrics/services/qualtrics-config.service.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+import { WindowRef } from '../../window';
+import { QualtricsConfig } from '../config/qualtrics-config';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class QualtricsConfigService {
```
[the remark this answers is not in the document]
Comment (bgambocjaviniar) [reply projects/core/src/qualtrics/services/qualtrics-config.service.ts]: Will be called QualtricsLoader as per Tobias request!

**Pull request closed without merging**